### ISSUE TITLE
reverseButtons and imageClass stays set across alerts

### ISSUE THREAD

**Issue by birjolaxew** [limonte/sweetalert2#245]
If an alert is fired with `reverseButtons: true` set, all following alerts will also have this setting, regardless of their actual setting.

[JSFiddle link.](https://jsfiddle.net/qzds24jn/1/)

Reproduced in Chrome, Firefox and IE11.

**Comment by birjolaxew** [comment]
Since the modal is reused, the moving done by [sweetalert2.js#L460](https://github.com/limonte/sweetalert2/blob/master/src/sweetalert2.js#L460) is never undone. We should either add a reverse case, or start recreating the alert content on each display.

I have tested the rest of the properties [(JSFiddle)](https://jsfiddle.net/yL4ddmoc/); `imageClass` also shows this behavior. I'd suggest just generating the modal all over again instead of having to think of this rather obscure bug every time we add a property. I'll wait for others' opinions before implementing anything though.

**Comment by limonte** [comment]
@birjolaxew 

> I have tested the rest of the properties (JSFiddle); imageClass also shows this behavior. I'd suggest just generating the modal all over again instead of having to think of this rather obscure bug every time we add a property. I'll wait for others' opinions before implementing anything though.

Yeah, I totally agree with you, but let's keep it for now in the old way: by adding `} else {`:

```js
if (params.reverseButtons) {
  $confirmButton.parentNode.insertBefore($cancelButton, $confirmButton);
} esle {
  …
}
```

**Comment by limonte** [comment]
I added tests and fixed the resetting of both parameters in [v4.1.5](https://github.com/limonte/sweetalert2/releases/tag/v4.1.5)
